packages/cli/src/modules/mcp/mcp.settings.service.ts

```diff
@@ -5,17 +5,22 @@ const KEY = 'mcp.access.enabled';
 
 @Service()
 export class McpSettingsService {
+	cacheIsEnabled: boolean | null = null;
```

Comment:
This in-memory cache can cause issues in multi-main. You should use the cacheService instead. Is there really a need for this?

Reply:
The only reason I did this it's because I do not want the module settings endpoint to make a query to the DB everytime we reload the page for no reason. This value rarely changes.